File: codeforces/problem_A_282/cpp/main.cpp
#include <iostream>
#include <string>
using namespace std;

int solution(int sum){
    string operation;
    cin >> operation;

    if (operation == "++X"){
        ++sum;
    }
    else if (operation == "X++"){
        sum++;
    }
    else if (operation == "--X"){
        --sum;
    }
    else if (operation == "X--"){
        sum--;
    }else{
    }
    return sum;
}

int main()
{
    int test_cases;
    cin >> test_cases;
    int sum = 0;
    while(test_cases--){
        sum = solution(sum);
    }
    cout << sum << endl;
    return 0;
}

File: codeforces/problem_A_282/python/main.py
def solution(sum):
    operation = input()
    if (operation == "++X"):
        sum += 1
    elif (operation == "X++"):
        sum += 1
    elif (operation == "--X"):
        sum -= 1
    elif (operation == "X--"):
        sum -= 1
    else:
        pass
    return sum

test_case = int(input())
sum = 0
while test_case > 0:
    sum = solution(sum)
    test_case -= 1

print(sum)


